io.github.MakovWait.Godots.json

```diff
@@ -0,0 +1,108 @@
+{
+    "id" : "io.github.MakovWait.Godots",
+    "runtime" : "org.freedesktop.Sdk",
+    "runtime-version" : "23.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk11",
+        "org.freedesktop.Sdk.Extension.dotnet7"
+    ],
+    "command": "godots",
+    "build-options": {
+        "append-path": "/usr/lib/sdk/dotnet7/bin",
+        "append-ld-library-path": "/usr/lib/sdk/dotnet7/lib",
+        "env": {
+            "PKG_CONFIG_PATH": "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet7/lib/pkgconfig"
+        }
+    },
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=x11",
+        "--device=all",
+        "--socket=pulseaudio",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--env=DOTNET_ROOT=/app/lib/dotnet",
+        "--env=DOTNET_CLI_TELEMETRY_OPTOUT=true",
+        "--env=DOTNET_NOLOGO=true"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "openjdk",
+            "buildsystem": "simple",
+            "build-commands": [
+                "/usr/lib/sdk/openjdk11/install.sh"
+            ]
+        },
+        {
+            "name": "dotnet",
+            "buildsystem": "simple",
+            "build-commands": [
+                "/usr/lib/sdk/dotnet7/bin/install-sdk.sh",
+                "cp -r /usr/lib/sdk/dotnet7/lib/packs/ /app/lib/dotnet/"
+            ]
+        },
+        {
+            "name" : "Godots",
+            "builddir" : true,
+            "buildsystem" : "simple",
+            "build-commands": [
+                "unzip Godot_v4.1.1-stable_linux.x86_64.zip",
+                "unzip Godot_v4.1.1-stable_export_templates.tpz",
+                "mkdir ./home",
+                "mkdir -p ./home/.local/share/godot/export_templates/4.1.1.stable",
+                "mv templates/* ./home/.local/share/godot/export_templates/4.1.1.stable",
+                "HOME=/run/build/Godots/home ./Godot_v4.1.1-stable_linux.x86_64 --headless --export-release 'Linux/X11' ./godots-bin",
+                "install -D -m755 ./godots-bin /app/bin/godots-bin",
+                "install -D -m755 ./godots.sh /app/bin/godots",
+                "desktop-file-edit --set-icon=$FLATPAK_ID $FLATPAK_ID.desktop",
```

Comment:
why not making it right in the file instead? Since it doesn't come from upstream.

Reply:
That's possible too, but why not just include it as a separate file for now, since it would be easier I imagine?

Reply:
Since you include the file, just make it right. desktop-file-edit is for when the file come from somewhere.

And then submit it for inclusion upstream, a desktop app without a .desktop is a bug. It's a 20+ year old spec.